x-pack/plugins/lens/public/xy_visualization/xy_config_panel.tsx

```diff
@@ -191,25 +234,72 @@ export function XyToolbar(props: VisualizationToolbarProps<State>) {
       : !state?.legend.isVisible
       ? 'hide'
       : 'show';
+
+  const valueLabelsVisibilityMode = state?.valueLabels || 'hide';
+
+  const isValueLabelsDisabled = hasNonBarSeries || !hasBarNotStacked || isHistogramSeries;
+
   return (
     <EuiFlexGroup gutterSize="m" justifyContent="spaceBetween">
       <EuiFlexItem>
         <EuiFlexGroup gutterSize="none" responsive={false}>
-          <TooltipWrapper
-            tooltipContent={i18n.translate('xpack.lens.xyChart.fittingDisabledHelpText', {
-              defaultMessage: 'This setting only applies to line and area charts.',
+          <ToolbarPopover
+            title={i18n.translate('xpack.lens.xyChart.valuesLabel', {
+              defaultMessage: 'Values',
             })}
-            condition={!hasNonBarSeries}
+            type="values"
+            groupPosition="left"
+            buttonDataTestSubj="lnsMissingValuesButton"
           >
-            <ToolbarPopover
-              title={i18n.translate('xpack.lens.xyChart.valuesLabel', {
-                defaultMessage: 'Values',
-              })}
-              isDisabled={!hasNonBarSeries}
-              type="values"
-              groupPosition="left"
-              buttonDataTestSubj="lnsMissingValuesButton"
+            <EuiFormRow
+              display="columnCompressed"
+              label={
+                <TooltipWrapper
+                  tooltipContent={i18n.translate(
+                    'xpack.lens.xyChart.valuesStackedDisabledHelpText',
+                    {
+                      defaultMessage:
+                        'This setting can only be used on categorical, un-stacked bar charts',
```

Comment:
Assuming my previous suggestion to conditionally hide this field is applied, we'll need to update this tooltip. Additionally, as a novice Lens user, it took me a few moments to understand what I had to do to my bar chart in order to be allowed to enable this setting. I'm wondering if verbiage such as this might make it clearer: `This setting may be modified on categorical bar charts, but not histograms.`

Someone else could likely better wordsmith this, but I think the notice that this is not for histogram-oriented bar charts makes it a bit more obvious. Thoughts?

Reply:
Agree on the wording problem, but perhaps it could be addressed in a separate PR?

Reply:
I'm fine if you would prefer to update this text in a separate PR.